### PR TITLE
Check existing projects while adding a new one

### PR DESCRIPTION
**Pull request by pypingou**
Fixes https://github.com/fedora-infra/anitya/issues/85

### CODE REVIEW

**Comment by puiterwijk** [comment]
Could you maybe put the html already in the template but then have it enabled (unhidden) by javascript, rather than creating the entire block in JS itself?
Makes it easier to modify the template I think.

:+1:.

File: anitya/templates/project_new.html [hunk 1]
```diff
@@ -115,11 +125,52 @@
 
   };
 
+  function show_project(_url, type) {
```
Comment (puiterwijk) [inline anitya/templates/project_new.html]: Maybe rename to show_similar_project?

Comment (pypingou) [inline anitya/templates/project_new.html]: sure why not, but it's not used anywhere else than this template/page

**Comment by puiterwijk** [comment]
Looks good to me. :shipit:

**Pull request merged**